packages/circuit/quri_parts/circuit/circuit_parametric.py

```diff
@@ -95,6 +75,34 @@ def get_mutable_copy(self) -> "MutableUnboundParametricQuantumCircuitProtocol":
         """
         ...
 
+    @abstractmethod
+    def primitive_circuit(self) -> "UnboundParametricQuantumCircuitProtocol":
+        r"""Returns the parametric circuit where each gate has an independent
+        parameter.
+
+        Note that some parametric circuit,
+        e.g. :class:`LinearMappedUnboundParametricQuantumCircuit`, can have non-trivial
+        mapping of the parameters. In this "primitive circuit", however,
+        gate parameters are treated as independent,
+        even if those in the original circuit depend on the same parameters.
+        For example, if the parametric circuit is defined as:
+
+        .. math::
+        \begin{align}
+            U_1(f(\theta_1, \theta_2)) U_2(g(\theta_1, \theta_2))
+        \end{align}
+
+        the primitive circuit should be as the following:
+
+        .. math::
+        \begin{align}
+            U_1(\psi_1) U_2(\psi_2)
+        \end{align}
```

Comment:
Math equations are not rendered correctly.
https://deploy-preview-37--quri-parts.netlify.app/quri_parts/circuit/quri_parts.circuit.circuit_parametric.html#quri_parts.circuit.circuit_parametric.UnboundParametricQuantumCircuitProtocol.primitive_circuit
Probably an extra indent should be added to each line inside `.. math::` directive.
```suggestion
        .. math::
            \begin{align}
                U_1(f(\theta_1, \theta_2)) U_2(g(\theta_1, \theta_2))
            \end{align}

        the primitive circuit should be as the following:

        .. math::
            \begin{align}
                U_1(\psi_1) U_2(\psi_2)
            \end{align}
```

Reply:
fixed